src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/events/OutboundEvents.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.court.events
+import java.time.LocalDateTime
+
+enum class OutboundEvent(val eventType: String) {
+  VIDEO_LINK_BOOKING_MIGRATE("whereabouts-api.videolink.migrate") {
+    override fun event(additionalInformation: AdditionalInformation) =
+      OutboundHMPPSDomainEvent(
+        eventType = eventType,
+        additionalInformation = additionalInformation,
+        description = "A video link booking has been identified for migration",
+      )
+  },
+  ;
+
+  abstract fun event(additionalInformation: AdditionalInformation): OutboundHMPPSDomainEvent
+}
+
+interface AdditionalInformation
```

Comment:
Possibly a little overkill with the interfacing.  There will only be one event then all of this code will be deleted? meh

Reply:
Part of the copy/paste - just a quick way.